src/main/java/org/jabref/model/util/FileHelper.java

```diff
@@ -22,9 +23,27 @@
 import org.apache.tika.mime.MimeType;
 import org.apache.tika.mime.MimeTypeException;
 import org.apache.tika.parser.AutoDetectParser;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class FileHelper {
-
+    /**
+     * MUST ALWAYS BE A SORTED ARRAY because it is used in a binary search
+     */
+    // @formatter:off
+    private static final int[] ILLEGAL_CHARS = {
+            0, 1, 2, 3, 4, 5, 6, 7, 8, 9,
+            10, 11, 12, 13, 14, 15, 16, 17, 18, 19,
+            20, 21, 22, 23, 24, 25, 26, 27, 28, 29,
+            30, 31, 34,
+            42,
+            58,
+            60, 62, 63,
+            123, 124, 125
+    };
+    // @formatter:on
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FileHelper.class);
     /**
```

Comment:
Please add an empty line to separate the JavaDoc comment from the variable declaration before